docs/source/app/holography.rst

```diff
@@ -0,0 +1,195 @@
+Off-axis electron holography
+============================
+
```

Comment:
Hi @woozey, the additions should be versioned, both in the code and documentation. You can add this line: 

`.. versionadded:: 0.3`